pages/s/[org]/[repo]/index.tsx

```diff
@@ -96,19 +97,17 @@ export default function RepoPage({ repoData, ogImageUrl }: RepoPageProps) {
   const isMobile = useMediaQuery("(max-width: 576px)");
   const avatarUrl = getAvatarByUsername(repoData.full_name.split("/")[0], 96);
   const [lotteryState, setLotteryState] = useState<"lottery" | "yolo">("lottery");
-  const [yoloHideBots, setYoloHideBots] = useState(
-    router.query.hideBots ? (router.query.hideBots === "true" ? true : false) : false
-  );
+  const yoloHideBots = router.query.hideBots === "true";
   const [isAddToWorkspaceModalOpen, setIsAddToWorkspaceModalOpen] = useState(false);
   const range = (router.query.range ? Number(router.query.range) : 30) as Range;
   const tabList = [
     { name: "Overview", path: "" },
     { name: "Contributors", path: "contributors" },
   ];
 
-  useEffect(() => {
-    router.push({ query: { ...router.query, hideBots: yoloHideBots } });
-  }, [yoloHideBots]);
+  function setYoloHideBots(value: boolean) {
+    setQueryParams({ hideBots: `${value}` });
```

Comment:
`setQueryParams` is pushing to the Next.js router like in the code prior to this change, but it defaults to maintaining the scroll position.

https://github.com/open-sauced/app/blob/86be85ceec2a31a85530d9b8dd96c147bd20c3bb/lib/utils/query-params.ts#L11-L31